kogito-maven-plugin/src/main/java/org/kie/kogito/maven/plugin/util/MojoUtil.java

```
@@ -53,7 +53,7 @@
         mavenProject.setArtifactFilter(new CumulativeScopeArtifactFilter(Arrays.asList("compile", "runtime")));
         for (final Artifact artifact : mavenProject.getArtifacts()) {
             final File file = artifact.getFile();
-            if (file != null) {
+            if (file != null && file.isFile()) {
```

When removing the lifecycle-mapping plugin to ignore errors in Eclipse from the kogito-examples parent module, the maven plugin was trying to unzip the XXX/classes folders and was raising an error because it's a folder, not a zip file. This check fixed the issue as looking at how this file is used, it's expected to be a zip file.